components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -204,6 +205,12 @@ function OrgSettingsForm(props: { org?: OrganizationInfo }) {
                 // handleUpdateTeamSettings({ defaultWorkspaceImage });
             }}
         >
+            {props.org && (
+                <InputField label="Organization ID">
+                    <InputWithCopy value={props.org.id} tip="Copy Org ID" />
```

Comment:
**nitpick**: Suggestion to use the full description and on the internal shorthand we use for organizations.

```suggestion
                    <InputWithCopy value={props.org.id} tip="Copy Organization ID" />
```